cmd/posix.go

```diff
@@ -1182,8 +1182,8 @@ func (s *posix) CreateFile(volume, path string, fileSize int64, r io.Reader) (er
 			return err
 		}
 	}
-	defer w.Sync() // Sync before close.
 	defer w.Close()
+	defer w.Sync() // Sync before close.
```

Comment:
Comment needs to be changes to `// Sync after close.`

Reply:
@kannappanr It is `Sync before close`. `defer` statement evaluates in LIFO order. So, the order in which the statements appear in code is reverse of the order of evaluation.

Reply:
ok. Makes sense.